doc/articles/includes/additional-windows-setup-inline.md

```diff
@@ -0,0 +1,18 @@
+In order to run Skia+GTK heads, you will need to make sure to install the [GTK3 runtime](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases).
```

Comment:
@jeromelaban not sure this is still needed for the Setup for Skia Desktop projects lately.
Is having a Windows tab still relevant here, please?
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-vs-2022.md?plain=1#L60
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-rider.md?plain=1#L49
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/common-issues-skia.md?plain=1#L18
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L24

Reply:
You can remove the Windows tabs on these pages @LProulx86. 
Make sure to remove the tab part if there is only one tab left for some pages.